app/validation/validators.py

```diff
@@ -130,7 +127,7 @@ def __init__(self, max_decimals=0, messages=None):
         self.messages = messages
 
     def __call__(self, form, field):
-        data = field.raw_data[0].replace(numbers.get_group_symbol(DEFAULT_LOCALE), '')
+        data = field.raw_data[0].replace(numbers.get_group_symbol(DEFAULT_LOCALE), '').replace(' ', '')
```

Comment:
Why are we removing spaces?

Reply:
if the user has entered a space by accident

Reply:
Is that a requirement?
If it is shouldn't we do that for all inputs?

Reply:
If so it should probably be a documented feature in its own PR?

Reply:
if its not there and user enters a space i believe they just get a 5xx, we wouldn't want that

Reply:
Yeah I think it's reasonable to strip spaces when parsing a numeric field. Even python does this:

```
>>> int(' 2     ')
2
```

Reply:
If python does it why do we need to?

Reply:
So if you run `test_numbers` and enter a figure with 2 DP into the decimal field and put a trailing space you get the Decimal error, this just eliminates that for our user